addStyles.js

```diff
@@ -35,6 +36,9 @@ module.exports = function(list, options) {
 	// By default, add <style> tags to the bottom of <head>.
 	if (typeof options.insertAt === "undefined") options.insertAt = "bottom";
 
+	// Fix urls defaults to false
+	if (typeof options.fixUrls === "undefined") options.fixUrls = false;
```

Comment:
There was a discussion in a previous PR about this. It seems like it makes sense to enable this by default. Otherwise we get broken CSS and no advice from Webpack on how to fix it.

Just to be safe and not break previous workarounds we could add few checks: 
1. that sourceMaps are enabled 
2. `output.publicPath` is not set